rust/cyclotron-fetch/src/config.rs

```diff
@@ -32,22 +32,39 @@ pub struct Config {
     #[envconfig(default = "false")]
     pub allow_internal_ips: bool,
 
-    pub worker_id: Option<String>,              // Default to a UUID
-    pub job_poll_interval_seconds: Option<u32>, // Defaults to 1
-    pub concurrent_requests_limit: Option<u32>, // Defaults to 1000
-    pub fetch_timeout_seconds: Option<u32>,     // Defaults to 30
-    pub max_retry_attempts: Option<u32>,        // Defaults to 10
-    pub queue_served: Option<String>,           // Default to "fetch"
-    pub batch_size: Option<usize>,              // Defaults to 1000
-    pub max_response_bytes: Option<usize>,      // Defaults to 1MB
-    pub retry_backoff_base_ms: Option<u32>,     // Defaults to 4000
+    #[envconfig(default = "default_worker_id")]
+    pub worker_id: String,
+
+    #[envconfig(default = "1")]
+    pub job_poll_interval_seconds: i64,
+
+    #[envconfig(default = "1000")]
+    pub concurrent_requests_limit: u32,
+
+    #[envconfig(default = "30")]
+    pub fetch_timeout_seconds: i64,
+
+    #[envconfig(default = "10")]
+    pub max_retry_attempts: u32,
+
+    #[envconfig(default = "fetch")]
+    pub queue_served: String,
+
+    #[envconfig(default = "1000")]
+    pub batch_size: usize,
+
+    #[envconfig(default = "1000000")]
+    pub max_response_bytes: usize,
+
+    #[envconfig(default = "4000")]
+    pub retry_backoff_base_ms: i64,
+}
+
+#[allow(dead_code)]
```

Comment:
rustc annoyingly doesn't understand that this is used by the annotation. Oh well.